libraries/AP_RangeFinder/RangeFinder.h

```diff
@@ -48,7 +48,8 @@ class RangeFinder
         RangeFinder_TYPE_LWSER  = 8,
         RangeFinder_TYPE_BEBOP  = 9,
         RangeFinder_TYPE_MAVLink = 10,
-        RangeFinder_TYPE_LEDDARONE = 12
+        RangeFinder_TYPE_LEDDARONE = 12,
```

Comment:
By the way, I think it's ok to use "11".  I'm pretty sure we skipped 11 because I was originally planning to use it for the Lightware SF40c but we ended up putting it into a new AP_Proximity library instead.  Anyway, using lucky 13 is also ok.